spec/swagger/definitions/hand_coded_paths.json

```diff
@@ -272,74 +272,110 @@
+            "application/vnd.api+json": {
+              "schema": {
+                "$ref": "put_move_responses.json#/200"
+              }
+            }
+          }
         },
-        {
-          "name": "filter[police_national_computer]",
-          "in": "query",
-          "description": "Filters results to only include people identified by that police_national_computer. Either filter[police_national_computer] or filter[prison_number] or both filters are required.",
-          "schema": {
-            "type": "string",
-            "example": "07/1435713R"
-          },
-          "format": "string",
-          "required": false
+        "400": {
+          "description": "bad request",
+          "content": {
+            "application/vnd.api+json": {
+              "schema": {
+                "$ref": "put_move_responses.json#/400"
+              }
+            }
+          }
         },
-        {
-          "name": "filter[prison_number]",
-          "in": "query",
-          "description": "Filters results to only include people identified by that prison_number. Either filter[police_national_computer] or filter[prison_number] or both filters are required.",
-          "schema": {
-            "type": "string",
-            "example": "G1234UT"
-          },
-          "format": "string",
-          "required": false
+        "401": {
+          "description": "unauthorized",
+          "content": {
+            "application/vnd.api+json": {
+              "schema": {
+                "$ref": "put_move_responses.json#/401"
+              }
+            }
+          }
         },
-        {
-          "name": "page",
-          "in": "query",
-          "description": "The page of records to return, defaults to 1 (the first page)",
-          "schema": {
-            "type": "integer"
-          },
-          "example": 1
+        "404": {
+          "description": "resource not found",
+          "content": {
+            "application/vnd.api+json": {
+              "schema": {
+                "$ref": "put_move_responses.json#/404"
+              }
+            }
+          }
         },
-        {
-          "name": "per_page",
-          "in": "query",
-          "description": "Number of records to return in a single response, defaults to 20, maximum value is 100.",
-          "schema": {
-            "type": "integer"
-          },
-          "example": 1
+        "415": {
+          "description": "invalid media type",
+          "content": {
+            "application/vnd.api+json": {
+              "schema": {
+                "$ref": "put_move_responses.json#/415"
+              }
+            }
+          }
+        },
+        "422": {
+          "description": "unprocessable entity",
+          "content": {
+            "application/vnd.api+json": {
+              "schema": {
+                "$ref": "put_move_responses.json#/422"
+              }
+            }
+          }
         }
+      }
+    },
+    "delete": {
```

Comment:
Can this endpoint and associated definition be deleted now?

Reply:
@smoothcontract yes, I think it probably should be - but would prefer to do that as separate PR for clarity